src/main/kotlin/com/avail/descriptor/methods/MacroDefinitionDescriptor.kt

```diff
@@ -6,13 +6,13 @@
  * Redistribution and use in source and binary forms, with or without
  * modification, are permitted provided that the following conditions are met:
  *
- *  * Redistributions of source code must retain the above copyright notice, this
- *    list of conditions and the following disclaimer.
+ *  * Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
  *
- *  * Redistributions in binary form must reproduce the above copyright notice, this
- *    list of conditions and the following disclaimer in the documentation
+ *  * Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
  *    and/or other materials provided with the distribution.
- *
+*
```

Comment:
Some of these copyrights are now mangled on this line.  Don't fix it on this pass, but eventually we should blindly replace anything that starts/ends the same as our copyright with the canonical form (and 1993-2020).

Reply:
They are all fixed now.